server/plugin/oauth.go

```diff
@@ -26,25 +27,25 @@ func (p *Plugin) httpOAuth2Connect(w http.ResponseWriter, r *http.Request) {
 func (p *Plugin) httpOAuth2Complete(w http.ResponseWriter, r *http.Request) {
 	code := r.URL.Query().Get("code")
 	if code == "" {
-		http.Error(w, "missing authorization code", http.StatusBadRequest)
+		p.handleAPIError(w, &serializer.APIErrorResponse{StatusCode: http.StatusBadRequest, Message: "missing authorization code"})
```

Comment:
we have used error messages starting with the capital letters above. check everywhere